wagtailio/static/sass/layout/_footer.scss

```diff
@@ -174,4 +174,35 @@
         }
 
     }
+
+    &__survey-link {
+        background-color: $color--teal;
+        position: fixed;
+        bottom: -2px;
+        right: 2rem;
+        padding: 12px 16px 10px;
```

Comment:
Suggestion: Use `inset` here.

Reply:
will leave that as a follow up. don't have the time to tinker with it